yoeo/utils/utils.py

```diff
@@ -41,12 +43,30 @@ def to_cpu(tensor):
     return tensor.detach().cpu()
 
 
-def load_classes(path):
+def load_classes(path: str) -> List[str]:
     """
     Loads class labels at 'path'
     """
+    if _is_yaml_file(path):
+        return _load_classes_from_yaml_file(path)
+    else:
+        return _load_classes_from_non_yaml_file(path)
```

Comment:
Do you want to keep the non yaml `.names` file? Especially if we also add names for the segmentation classes it could be easier to drop non-yaml support.

Reply:
Well, right now, `_load_classes_from_yaml_file()` is only loading the detection class names. I figured the segmentation class names are not strictly necessary for the YOEO repo since there are no such class names in the .name file. The idea was to allow the use of a yoeo_names.yaml file (required by YOEOHandler) without breaking backwards compatibility. But we can surely drop non-yaml support if you wish.

Reply:
non-yaml support is dropped